### PR TITLE
Fix various resizing bugs

### LINKED ISSUE #124: Cannot resize screenshot below certain height
- [x] I have searched open and closed issues for duplicates
- [x] I am submitting a bug report for existing functionality that does not work as intended
- [x] This isn't a feature/enhancement request or a discussion topic

----------------------------------------

### Bug description
A screenshot, once taken, cannot be resized below a certain height.

### Steps to reproduce the bug
- Take a screenshot.
- Try to resize the image such that it has no height.

**Actual result:** You cannot reduce the height below a certain point.
**Expected result:** Screenshot height should be allowed to be smaller.

### Version info
**Shotr Version:** 3.1.5

### PR DESCRIPTION
**Pull request by Tursko**
This PR fixes #124 and #133.

### CODE REVIEW

**Comment by zacbre** [comment]
man some of these comments are 7 or 8 years old

File: src/Shotr.Ui/Forms/ScreenshotForm.cs [hunk 1]
```diff
@@ -422,68 +423,72 @@ void ScreenshotForm_MouseMove(object sender, MouseEventArgs e)
...
+                            if (proposedLeftX < currentRightX)
+                            {
                                 _x.X += output;
+                            }
                             else
-                                _x.X = currentRightX - 10;
+                            {
+                                _x.X = currentRightX - _MinSize;
+                            }
                             // Prevent width from going into the negatives, this can cause overlap. No smaller than 10px.
```
Comment (Tursko) [inline src/Shotr.Ui/Forms/ScreenshotForm.cs:441]: I need to fix this comment.

**Comment by Tursko** [comment]
Ready for review :)

**Review by zacbre — APPROVED** [review]
Dude this change is awesome, wow. Great find and great fix!! Approved!!

**Pull request merged**